README.md

```diff
@@ -157,6 +157,33 @@ logs:
 
     Help > Show Logs
 
+### NeoVim Setup
+
+**Note**: While these instructions are for NeoVim, they should be transferable
+to Vim.
+
+For [vim-plug](https://github.com/junegunn/vim-plug) users, install
+[Coc](https://github.com/neoclide/coc.nvim) and its plugin
+[coc-erlang_ls](https://github.com/hyhugh/coc-erlang_ls) enter:
```

Comment:
Nit: enter -> by entering

Reply:
it makes sense if you read past the block of code, but that structure is confusing, will reorganise